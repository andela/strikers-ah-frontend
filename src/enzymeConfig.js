import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import 'jest-localstorage-mock';

export default Enzyme.configure({
  adapter: new Adapter(),
});
