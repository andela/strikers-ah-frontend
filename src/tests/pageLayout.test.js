import React from 'react';
import { shallow } from 'enzyme';
import '../enzymeConfig';
import Layout from '../components/common/pageLayout';

describe(' Test <Layout/>', () => {
  it('render <Layout/> without crashing', () => {
    const wrapper = shallow(<Layout />);
    expect(wrapper).toMatchSnapshot();
  });
});
