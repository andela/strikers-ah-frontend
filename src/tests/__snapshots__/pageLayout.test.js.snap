// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Test <Layout/> render <Layout/> without crashing 1`] = `ShallowWrapper {}`;
